src/styles/Container.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+    background-color: #fff;
+    width: 100vw;
+    height: 100vh;
+    @media screen and (min-width: 600px) {
```

Comment:
```suggestion
    @media screen and (min-width: 768px) {
```
상황에 따라 적절한 값을 사용해야 겠지만, 반응형 제작 시 참고하는 모바일 디바이스 표준 width 값은 <= 768px 입니다.